.travis.yml

```diff
@@ -12,6 +12,7 @@ branches:
   - master
   - develop
   - /branch-.*$/
+  - /fix\/part3-rest-api-deletion-endpoint$/
```

Comment:
```suggestion
  - /feature.*$/
```

Reply:
Yup, but this branch is `fix` and not feature no tests will not run automatically. I think that we should not discard all changes is travis.yml when merging in develop.

Reply:
The builds are triggered based on the name of the branches, so if the target branch contains `feature` for example, it will work, even though your source branch doesn't